### PR TITLE
feat(Modal): Disclaimer modal added

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**

### CODE REVIEW

**Comment by bobachef** [comment]
Will merge after fade in effect added so modal isn't so sudden

File: assets/js/connectWallet.js [hunk 1]
```diff
@@ -71,6 +69,13 @@ async function fetchAccountData() {
 
   document.querySelector("#selected-account").textContent = selectedAccount;
 
+  // Short account
+  const firstPart = selectedAccount.slice(0 ,4);
+  const secondPart = selectedAccount.slice(selectedAccount.length - 4, selectedAccount.length);
+  const shortAccount = firstPart + '...' + secondPart;
+  document.querySelector("#shortAddress").textContent = shortAccount;
```
Comment (bobachef) [inline assets/js/connectWallet.js:76]: consider using string interpolation

const shortAccount = `${firstPart}...${secondPart}`

Comment (Shaharyar-saleem) [reply assets/js/connectWallet.js:76]: used

Comment (bobachef) [reply assets/js/connectWallet.js:76]: you just haven't pushed right?

Comment (bobachef) [reply assets/js/connectWallet.js:76]: wait until you've committed and pushed and then you can resolve this

**Pull request merged**